client/src/extension.ts

```diff
@@ -563,9 +563,11 @@ export function realActivate (context: ExtensionContext): void {
                   changeProcessCWD = !!entry.changeProcessCWD
                 }
                 if (directory != null) {
-                  if (path.isAbsolute(directory)) {
-                    directory = directory
-                  } else if (workspaceFolderPath != null && directory != null) {
+                  if (
+                    !path.isAbsolute(directory) &&
+                    workspaceFolderPath != null &&
+                    directory != null
+                  ) {

```

Comment:
This code change is not the same as what was here before. Now if `path.isAbsolute(directory)` is true, then the `else` case will be executed which sets `directory` to `undefined`. You could fix this by changing the last `else` case to `else if (!path.isAbsolute(directory)) {`

Reply:
Indeed, good catch!
I fixed it in the last commits, I just pushed. :+1: